src/dom.js

```diff
@@ -16,6 +16,35 @@
 
 
 /**
+ * Whether the element is currently contained in the DOM. Polyfills
+ * `document.contains()` method when necessary.
+ * See https://developer.mozilla.org/en-US/docs/Web/API/Node/contains
+ * @param {!Document} doc
+ * @param {!Element} element
+ * @return {boolean}
+ */
+export function documentContains(doc, element) {
+  if (!doc.contains) {
+    return documentContainsPolyfillInternal_(doc, element);
+  }
+  return doc.contains(element);
```

Comment:
I don't know that yet, unfortunately - i have no way to test it. We are kind of getting into more obscure cases here. If it fails again - I'll have to do more polyfiling.